### DISCUSSION TITLE
We are having issue with API Unknown Error while sending inventory through API. Attaching the error screenshots for your reference

### DISCUSSION THREAD

**Discussion by retailtantra** [Sellbrite/Sellbrite-API#34] [Q&A]
We are having issue with API **Unknown Error** while sending inventory through API. Attaching the error screenshots for your reference
![1710774628556blob](https://github.com/Sellbrite/Sellbrite-API/assets/164185347/74462761-a317-4b2a-b8a5-184b0548ccd7)

**Comment by rengler-godaddy** [comment]
Hello, 

Would you be able to post the full request with the body params? Exclude any details that show your account token and secret key.
